docker/build.sh

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/env bash
+
+set -euxo pipefail
+
+# two `cd`'s so Git can find the root dir even if this script is called from
+# outside the repo
+cd $(dirname "${BASH_SOURCE[0]}")
```

Comment:
I think this is more portable.
Some people use zsh etc
```suggestion
cd -P -- "`dirname -- "$0"`"
```

(but perhaps those people are not trying to build isle :) )

Reply:
Most people have bash installed anyways, hence the bash shebang. But seems fine.